src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -155,7 +155,9 @@ export class QuestionsPageComponent implements OnInit {
       setTimeout(() => this.updateToolbarButtons(), 100)
     }
     if (this.questionsService.getIsNextAutomatic(event.type)) {
-      this.nextQuestion()
+      setTimeout(() => {
+        this.nextQuestion()
+      }, 20)
```

Comment:
is this to give a delay before moving to the next question? I am not sure a 20ms delay would be perceivable?

Reply:
I added this for debugging purposes, I've now removed it.